routing/routes.go

```diff
@@ -36,12 +36,30 @@ func NewRoutes() (r *Routes) {
 			"/api/status",
 			handlers.Root,
 		},
+		{
+			"IstioConfigList",
+			"GET",
+			"/api/namespaces/{namespace}/istio",
+			handlers.IstioConfigList,
+		},
+		{
+			"IstioConfigDetails",
+			"GET",
+			"/api/namespaces/{namespace}/istio/{object_type}/{object}",
+			handlers.IstioConfigDetails,
+		},
+		/*
+			TODO the /rules endpoint is now deprecated in favor of /istio
```

Comment:
do we need to deprecate instead of removing straight away? We're still the only consumers of our api after all

Reply:
I was writting a comment about it :-).
Yeah, I want to remove this, but before to apply the changes in the UI, to not break things until the UI is done.

In timeline,
- Add new endpoint in backend
- Modify UI with new feature
- Remove deprecated endpoints

In that case, we don't break the UI if this PR and the UI counter-part and not in at same time.

But yes, those endpoints are not useful anymore.

Reply:
+1. this is the way I have been doing things as well, so that the UI continues to work during back-end changes.  Have the back-end support old and new, update UI to new, remove old from back-end.